src/calculator/index.ts

```diff
@@ -185,13 +187,21 @@ export function buildLots(stockTransactions: StockTransaction[], eacHistory: EAC
         else if (sellToCoverTransaction !== undefined) {
             lots.push({
                 symbol: spaTransaction.symbol,
-                quantity: spaTransaction.quantity,
+                quantity: sellToCoverTransaction.rows.find(isSellToCoverHoldRow)?.sharesExercised || spaTransaction.quantity,
```

Comment:
So is the `sharesExercised` field sometimes different from the `quantity` field? If so then why? What does the two different numbers represent?